modules/azure/common.go

```diff
@@ -0,0 +1,39 @@
+package azure
+
+import "os"
+
+const (
+	// AzuSubID is an optional env variable custom to Terratest to designate a target Azure subscription ID
+	AzuSubID = "AZURE_SUB_ID"
+
+	// AzuResGroupName is an optional env variable custom to Terratest to designate a target Azure resource group
+	AzuResGroupName = "AZURE_RG_NAME"
```

Comment:
Minor nit: `AZURE_RES_GROUP_NAME` seems more clear than `AZURE_RG_NAME` (where I immediately thought, what's RG?)

Reply:
I figured anyone using azure would know this, but totally fair callout.  Will refactor.